api/services/userService.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const { userDao } = require('../models');
+
+const signIn = async (email, password) => {
+  const user = await userDao.getUserByEmail(email);
+
+  if (!user) {
+    const error = new Error("USER_IS_NOT_VALID")
+    error.statusCode = 401;
+    throw error;
+  }
+
+  const isMatched = await bcrypt.compare(password, user.password);
+  
+  if (!isMatched) {
+    const error = new Error("PASSWORD_IS_NOT_VALID")
```

Comment:
email이 틀렸는지, password가 틀렸는지 모르게 에러 메시지를 전달해 주세요.
```suggestion
    const error = new Error("USER_IS_NOT_VALID")
```